app/views/experience_stages/show.html.erb

```diff
@@ -91,15 +91,15 @@
               </div>
             </div>
 
-            <div ng-switch='theme_<%= theme.id %>_experiences' ng-init='theme_<%= theme.id %>_experiences=show_all'>
+            <div ng-switch='theme_<%= theme.id %>_experiences' ng-init='theme_<%= theme.id %>_experiences=false'>
```

Comment:
This was setting `$scope.theme_:id_experiences = $scope.show_all`. And `$scope.show_all` was being set to true as long as `window.location.hash` was not empty. This meant that upon clicking the Popular/Latest links, the page reloaded with all themes expanded (showing all associated experience cards).